### PR TITLE
Test/slight adjustments

### PR DESCRIPTION
**Pull request by scnale**
This merges end to end test transfer scripts for Solana and EVM into a single script.
It also adds an environment variable `FLIP_ROUTES` that allows you to request the reverse transfer to go back to the source chain for each test case specified.

### CODE REVIEW

File: deployment/test/test-transfer.ts [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+const usdcAddresses = {
+  OptimismSepolia: '0x5fd84259d66Cd46123540766Be93DFE6D43130D7',
+  Sepolia: '0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238',
+  Celo: ' 0x2F25deB3848C207fc8E0c34035B3Ba7fC157602B',
+  Solana: '4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU'
+};
+
+/**
+ * This script expects the following environment variables:
+ * - WALLET_KEY: private key of the wallet to use for sending transactions
```
Comment (mat1asm) [inline deployment/test/test-transfer.ts:51]: Do we need a different env var to be able to send from evm and from solana in a single run?

Comment (mat1asm) [reply deployment/test/test-transfer.ts:51]: nvm, we can play with the skip flag to as well

**Review by mat1asm — APPROVED** [review]

File: deployment/test/test-transfer.ts (hunk 1 above)
Comment (scnale) [reply deployment/test/test-transfer.ts:51]: Hm, yeah, we probably do. I forgot to check that.

Comment (scnale) [reply deployment/test/test-transfer.ts:51]: I mean, it is possible to use the same private key but it's a bit inflexible.

**Pull request merged**